shared-libs/transitions/test/unit/pregnancy_registration.js

```diff
@@ -379,7 +379,7 @@ describe('pregnancy registration with weeks since LMP', () => {
     return transition.onMatch({ doc: doc }).then(function(changed) {
       assert.equal(changed, true);
       assert.equal(doc.patient_id, undefined);
-      assert.equal(getMessage(doc), 'Invalid patient name.  Invalid LMP; must be between 0-40 weeks.');
+      assert.equal(getMessage(doc), 'Invalid LMP; must be between 0-40 weeks.  Invalid patient name.');
```

Comment:
This is technically broken backwards compatibility because the errors are returned in a different order. This is because instead of the "medic" validations being run separately, they are run at the same time as the pupil validations. I doubt anyone is relying on the order of the validation errors but it's worth pointing it out!